bundle/deploy/terraform/init.go

```diff
@@ -70,6 +70,22 @@ func (m *initialize) findExecPath(ctx context.Context, b *bundle.Bundle, tf *con
 	return tf.ExecPath, nil
 }
 
+// This function inherits some environment variables for Terraform CLI.
+func inheritEnvVars(env map[string]string) error {
+	// Include $HOME in set of environment variables to pass along.
+	home, ok := os.LookupEnv("HOME")
+	if ok {
+		env["HOME"] = home
+	}
+
+	// Include $TF_CLI_CONFIG_FILE to override terraform provider in development.
+	if config_file, ok := os.LookupEnv("TF_CLI_CONFIG_FILE"); ok {
+		env["TF_CLI_CONFIG_FILE"] = config_file
```

Comment:
Style: we use camelCase instead of snake_case (following the Go style guide).

Also see: https://go.dev/doc/effective_go#mixed-caps

Here, since the variable is scoped to the if block, you can use `val` or something opaque as well.